SonarQube.TeamBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -0,0 +1,191 @@
+﻿//-----------------------------------------------------------------------
```

Comment:
Most of the code in this class was copied from the SonarLintAnalyzerProvider. I'll call out the differences.